executor/common_bsd.h

```diff
@@ -193,7 +207,11 @@ struct tcp_resources {
 	uint32 ack;
 };
 
+#if defined(__FreeBSD__)
```

Comment:
We need `#if GOOS_freebsd` here. When we preprocess this in pkg/csource macros like `__FreeBSD__` are not defined. But `GOOS_*` are defined always. And theoretically we can generate freebsd reproducer on another OS.